src/format_text.rs

```diff
@@ -55,7 +55,7 @@ pub fn format_text(file_path: &Path, input_text: &str, config: &Configuration) -
       let printed = formatted.print()?;
       printed.into_code()
     }
-    Some("js" | "jsx" | "ts" | "tsx" | "cjs" | "mjs" | "cts" | "mts") => {
+    Some("js" | "jsx" | "ts" | "tsx" | "cjs" | "mjs" | "cts" | "mts" | "astro" | "svelte" | "vue") => {
```

Comment:
The code below formats js files, so won't this cause errors to surface for these file types? Maybe adding a test to this PR might help demonstrate the fix?

Reply:
This line below determines the JS syntax from the file path:
```rs
let Ok(syntax) = JsFileSource::try_from(file_path) else {
```
Which winds up here: https://github.com/biomejs/biome/blob/561b54c93625c9a83035562022af47237e6c00b2/crates/biome_js_syntax/src/file_source.rs#L300

Reply:
I'm down to add a test though : )

Reply:
Oh, interesting. For adding a test it would just be adding a file like this one: https://github.com/dprint/dprint-plugin-biome/blob/main/tests/specs/Json.txt (but making the top line have a .svelte extension) then running `cargo test`

Reply:
Also don't forget to add a thing for `json` and `jsonc` since it's fully supported in biome.

Reply:
https://github.com/dprint/dprint-plugin-biome/blob/3bdc77b02c4ea7a0908320dddf32788178c29baf/src/format_text.rs#L34